tests/test_utils/amazon_system_helpers.py

```diff
@@ -51,6 +54,27 @@ def provide_aws_s3_bucket(name):
 @pytest.mark.system("amazon")
 class AmazonSystemTest(SystemTest):
 
+    @staticmethod
+    def _region_name():
+        return os.environ.get("REGION_NAME")
+
+    @staticmethod
+    def _registry_id():
+        return os.environ.get("REGISTRY_ID")
+
+    @staticmethod
+    def _repository_name():
+        return os.environ.get("REPOSITORY_NAME")
+
+    @staticmethod
+    def _execution_role_arn():
+        return os.environ.get("EXECUTION_ROLE_ARN")
```

Comment:
```suggestion
    @property
    def _region_name():
        return os.environ.get("REGION_NAME")

    @property
    def _registry_id():
        return os.environ.get("REGISTRY_ID")

    @property
    def _repository_name():
        return os.environ.get("REPOSITORY_NAME")

    @property
    def _execution_role_arn():
        return os.environ.get("EXECUTION_ROLE_ARN")
```

Reply:
not sure, these are not getter and not returning `self._x`. the `@property` decorator is used for such things afaik.

also `GoogleSystemTest` class has same usage

Reply:
`@property` is not necessarily used to return `self._x`. IMHO, `@property` fits better here because the names and functionalities are pretty much attributes access here (though it is getting from the `ENV`).

I think this is more about a design decision, and better stay consistent across the codebase. I am not sure how other places are using this, so I think @mik-laj @potiuk @ashb can give better suggestions.

Reply:
Personally, I would like to improve this code a bit, but until it is run on CI, I refrained. We have too many static methods that could be independent methods. This makes it difficult to write tests that apply to many cloud providers. For now, I would prefer to keep consistency in the project, because it will facilitate future refactor.

Reply:
resolving this thread, reopen if you want me to change please. thank you all for review